composeApp/src/commonMain/kotlin/domain/EventsRepository.kt

```diff
@@ -0,0 +1,4 @@
+package domain
+
+interface EventsRepository {
```

Comment:
Could we add the methods/functions definitions?

```kotlin
fun getLastEvents(int offset = 0): List<Event> 
fun getEventInfo(String eventId): Event
```

Reply:
done